Automattic/wp-calypso
# Command Palette: Extract command palette to package

Related to:
- https://github.com/Automattic/dotcom-forge/issues/5493

## Proposed Changes
* Extracted the command palette to a package.
* We now support multiple command repositories.

## Testing Instructions



* Open the live link and check that there are no regressions when using the command palette.

## Pre-merge Checklist
